tests/functional/test_freeze.py

```diff
@@ -258,41 +259,31 @@ def test_freeze_mercurial_clone(script, tmpdir):
     _check_output(result, expected)
 
 
-@pytest.mark.network
 def test_freeze_bazaar_clone(script, tmpdir):
     """
     Test freezing a Bazaar clone.
 
     """
-
-    checkout_path = local_checkout(
-        'bzr+http://bazaar.launchpad.net/%7Edjango-wikiapp/django-wikiapp/'
-        'release-0.1',
-        tmpdir.join("cache"),
-    )
-    # bzr internally stores windows drives as uppercase; we'll match that.
+    try:
+        checkout_path = _create_test_package(script, vcs='bazaar')
+    except OSError as e:
+        pytest.skip('Invoking `bzr` failed: %s' % e)
```

Comment:
I'm not very familiar with py.test, but I guess the test will be marked as skipped if `_create_test_package` fails ?

I know that when I launch a test suite and I see a skipped test, I'm not going to check whether I could install something to avoid the test to pass.

It's not that hard to install bzr, so I'd be more in favor of a failing test that the developer can explicitly skip thanks to a `@pytest.mark.bzr`.